sql/mob_droplist.sql

```diff
@@ -33024,6 +33024,37 @@ INSERT INTO `mob_droplist` VALUES (4658,0,0,1000,2814,250); -- (Big Bang) Aptant
 INSERT INTO `mob_droplist` VALUES (4659,0,0,1000,11496,48); -- (Lizardtrap) Fenrir's Crown
 INSERT INTO `mob_droplist` VALUES (4660,0,0,1000,16035,41); -- (Goaftrap) Altdorf's Earring
 
+INSERT INTO `mob_droplist` VALUES (4661,0,0,1000,1628,104); -- (King Buffalo) Buffalo Hide
```

Comment:
It looks like you're taking these drop values from [ffxidb.com](http://ffxidb.com/zones/5/king-buffalo), which is good. But you should use the numbers from the TH0 column, when available, rather than the average column.  For example, Buffalo Hide would be 68 instead of 104.

Reply:
Understood I will make the change.